packages/dappmanager/src/modules/installer/runPackages.ts

```diff
@@ -54,13 +55,31 @@ export async function runPackages(
 
       // To clean-up changing multi-service packages, remove orphans
       // but NOT for core packages, which always have orphans
-      const removeOrphans = !pkg.isCore;
 
       log(pkg.dnpName, "Starting package... ");
-      await dockerComposeUp(pkg.composePath, {
-        removeOrphans,
-        timeout: pkg.dockerTimeout
-      });
+      const removeOrphans = !pkg.isCore;
+
+      if (!pkg.running) {
+        await dockerComposeUp(pkg.composePath, {
+          noStart: pkg.running,
+          removeOrphans,
+          timeout: pkg.dockerTimeout
+        });
+      } else if (pkg.running) {
+        const packageInfo = await listPackage(pkg);
```

Comment:
Move this logic to packages/dappmanager/src/modules/installer/getInstallerPackageData.ts